### PR TITLE
dwmapi.go: add definitions for various DWM APIs

### PR DESCRIPTION
**Pull request by dblohm7**
Updates https://github.com/tailscale/walk/issues/85

### CODE REVIEW

File: dwmapi.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	DWMWA_COLOR_NONE    uint32 = 0xFFFFFFFE
+	DWMWA_COLOR_DEFAULT uint32 = 0xFFFFFFFF
+)
+
+type DWMNCRENDERINGPOLICY int32
+
+const (
+	DWMNCRP_USEWINDOWSTYLE DWMNCRENDERINGPOLICY = 0
+	DWMNCRP_DISABLED       DWMNCRENDERINGPOLICY = 0
+	DWMNCRP_ENABLED        DWMNCRENDERINGPOLICY = 0
```
Comment (nickkhyl) [inline dwmapi.go]:
```suggestion
	DWMNCRP_DISABLED       DWMNCRENDERINGPOLICY = 1
	DWMNCRP_ENABLED        DWMNCRENDERINGPOLICY = 2
```

File: dwmapi.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+	DWMWA_CAPTION_COLOR                  DWMWINDOWATTRIBUTE = 35
+	DWMWA_TEXT_COLOR                     DWMWINDOWATTRIBUTE = 36
+	DWMWA_VISIBLE_FRAME_BORDER_THICKNESS DWMWINDOWATTRIBUTE = 37
+	DWMWA_SYSTEMBACKDROP_TYPE            DWMWINDOWATTRIBUTE = 38
+)
+
+const (
+	DWM_CLOAKED_APP       uint32 = 1
+	DWM_CLOAKED_SHELL     uint32 = 2
+	DWM_CLOAKED_INHERITED uint32 = 3
```
Comment (nickkhyl) [inline dwmapi.go]:
```suggestion
	DWM_CLOAKED_INHERITED uint32 = 0x4
```
Can you please also use hex notation for the preceding constants?

**Review by nickkhyl — APPROVED** [review]
LGTM with the const values fixed.

File: dwmapi.go (hunk 1 above)
Comment (dblohm7) [reply dwmapi.go]: Whoops! How sloppy of me!

**Pull request merged**